### ISSUE TITLE
Installing using singularity

### ISSUE THREAD

**Issue by Karimi-81** [shilpagarg/DipAsm#23]
I would like to install the "DipAsm" ; however it is not possible for me to use Docker on our servers directly. Therefore, we have to use docker file through singularity. I would be appreciate if you could provide any instruction that makes it possible to run the software using singularity.

**Comment by olechnwin** [comment]
I second this request. 
I think you only need to upload your docker image to docker hub then we can use singularity.

**Comment by lilinzhou** [comment]
I agree that most people who run a cluster without root would prefer to use singularity, or could we install it directly without using Docker or singularity, since DeepVariant can also be installed from source code.
